app/views/tags/feed.atom.builder

```diff
@@ -6,10 +6,10 @@ atom_feed do |feed|
     unless work.unrevealed?
       feed.entry work do |entry|
         entry.title work.title 
```

Comment:
Because at that point you have done the title in @works already which is cached so you don't gain anything and increase the complexity.